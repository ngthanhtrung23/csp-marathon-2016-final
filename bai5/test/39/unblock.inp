11233.
4.2.56
400.56
477756
...899
ZZ.8AA
