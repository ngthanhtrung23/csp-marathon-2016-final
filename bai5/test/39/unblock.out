45
6 U 1
Z R 1
4 D 2
0 L 1
2 D 1
3 L 1
5 U 1
7 R 2
2 D 2
0 R 1
4 U 1
Z L 1
2 D 1
7 L 2
5 D 1
3 R 1
1 R 1
4 U 2
7 L 1
8 U 3
7 R 1
4 D 1
1 L 1
3 L 1
5 U 1
7 R 2
2 U 1
A L 1
Z R 1
4 D 2
0 L 1
2 U 2
7 L 2
5 D 1
9 L 2
6 D 3
3 R 2
2 U 1
0 R 1
4 U 1
8 U 1
Z L 1
A L 1
5 D 2
0 R 3

